### DISCUSSION TITLE
broadcast see permission

### DISCUSSION THREAD

**Discussion by levarar** [Zrips/CMI#4033] [Feature Requests]
Hi, I want to give players the opportunity to turn off notification from broadcast, please add a permission for this action

**Comment by CloudeLecaw** [comment]
Instead if using a broadcast you can use a msg! To allplayers and set a permissioncheck before. `perm:cmi.seebroadcast! allPlayers! msg! [allPlayers] this is a nice broadcasts.`
Create all your broadcasts like that. 
Now just players with permission cmi.broadcast permission will see the broadcast.

If you want an option to turn on an off the broadcast for players as command simple create a customalias like informer on  and informee off  that execute the command as console and give ir take away the permission cmi.seebroadcast.
Use asConsole! for that before the permission give/take command. And for the playername use the placeholder [playerName].
The commandstructue for given and takenaway commands is based on your permission system ;)
 Also after the give/take permission command i recomment to use a secound command like a msg! to the player as a feedback meassage like "you have turned on/offf Server broadcasts for you"
You find more Informationen in the wiki - creating of customalias

**Comment by CloudeLecaw** [comment]
Hmm maybe i am wrong and thus is not working because sheduler are executed by console. That has all permission.
